src/content/docs/apm/agents/net-agent/other-installation/how-verify-checksum-net-agent-downloads.mdx

```diff
@@ -1,70 +1,69 @@
 ---
-title: How to verify the checksum of .NET Agent downloads
-tags:
-  - Agents
-  - NET agent
-  - Other installation
+title: How to verify the checksum of .NET agent downloads
 redirects:
   - /docs/agents/net-agent/other-installation/how-verify-checksum-net-agent-downloads
 freshnessValidatedDate: never
 ---
 
 Read about how to confirm the checksum for New Relic's .NET agent installers. If you need the hashes for our current releases, [download their SHA256 hashes from our download site](https://download.newrelic.com/dot_net_agent/latest_release/SHA256/).
 
-## Windows
+## Linux [#checksum-linux]
 
-On Windows, we recommended using either CertUtil or Get-FileHash to compute a file hash.
-
-### CertUtil
-
-This utility is available as part of Certificate Services and can be used to compute the hash of a file. This utility can be run from the Command Prompt or Powershell.
-
-To compute the hash, run:
-
-`CertUtil -hashfile YOUR_FILENAME SHA256`
-
-If successful, you'll get this response:
+On Linux, run:
 
 ```
-SHA256 hash of YOUR_FILENAME:
-5092fd52e40132a41ac06c320296cd9d63196d6de0fa6ce9b74fc4de1b3f9502
-CertUtil: -hashfile command completed successfully.
+sha256sum YOUR_FILENAME
 ```
 
-### Powershell: Get-FileHash [#getfilehash]
-
-The Get-FileHash cmdlet is included with Powershell version 5.1+ and can be used to compute the hash of a file.
-
-To compute the hash, run:
-
-`Get-FileHash YOUR_FILENAME Algorithm SHA256 | Format-List`
-
 If successful, you'll get this response:
 
 ```
-Algorithm : SHA256
-Hash      : 5092FD52E40132A41AC06C320296CD9D63196D6DE0FA6CE9B74FC4DE1B3F9502
-Path      : C:\Path\To\YOUR_FILENAME
+5092fd52e40132a41ac06c320296cd9d63196d6de0fa6ce9b74fc4de1b3f9502 YOUR_FILENAME
 ```
 
-## Linux
+## Windows [#checksum-windows]
 
-On Linux, we recommend using
+On Windows, we recommended using either `CertUtil` or `Get-FileHash` to compute a file hash.
 
-```
-sha256sum
-```
+<CollapserGroup>
+    <Collapser
+        id="certutil"
+        title="Command Prompt or Powershell: CertUtil"
+    >
+        This utility is available as part of Certificate Services and can be used to compute the hash of a file. This utility can be run from the Command Prompt or Powershell.
 
-to compute a file hash.
+        To compute the hash, run:
 
-### sha256sum [#she256sum]
+        ```
+        CertUtil -hashfile YOUR_FILENAME SHA256
+        ```
 
-To compute the hash, run:
+        If successful, you'll get this response:
 
-`sha256sum YOUR_FILENAME`
+        ```
+        SHA256 hash of YOUR_FILENAME:
+        5092fd52e40132a41ac06c320296cd9d63196d6de0fa6ce9b74fc4de1b3f9502
+        CertUtil: -hashfile command completed successfully.
+        ```
+    </Collapser>
+    <Collapser
+        id="getfilehash"
+        title="Powershell: Get-FileHash"
+    >
+        The `Get-FileHash` cmdlet is included with Powershell version 5.1+ and can be used to compute the hash of a file.
```

Comment:
You can use the `Get-FileHash` cmdlet included with Powershell version 5.1+ to compute the hash of a file.